.travis.yml

```diff
@@ -45,9 +45,12 @@ install:
   - make  # generate Python protobuf files and build shared libraries
 script:
   - tox
-  - poetry install -v
+  - poetry build -v
+  # export test dependencies from pyproject.toml, install them
+  - poetry export -f requirements.txt --dev -o requirements.txt && pip install -r requirements.txt
+  - pip install dist/*.whl
```

Comment:
will this install wheel locally? Will the test be run against this or poetry set up?

Reply:
Good point. I need to drop the `poetry run` bit.